Documentation/ceph-monitoring.md

```diff
@@ -81,11 +81,20 @@ You can find Prometheus Consoles here: https://github.com/ceph/cephmetrics/tree/
 A guide to how you can write your own Prometheus consoles can be found on the official Prometheus site here: https://prometheus.io/docs/visualization/consoles/.
 
 ## Prometheus Alerts
-To enable Prometheus based alerting for Ceph, run the following commands:  
+To enable prometheus alerts during cluster deployment, make following changes to `cluster.yaml`
+```YAML
+monitoring:
```

Comment:
A late note here, this new `monitoring` struct / section of the Ceph Cluster CRD has not been added to the Ceph Cluster CRD doc page.

Reply:
Will fix it.